src/pages/workspaces/workspace/analysis/Analyses.js

```diff
@@ -265,17 +264,18 @@ const Analyses = _.flow(
     const notebooks = _.filter(({ name }) => _.endsWith(`.${tools.Jupyter.ext}`, name), rawAnalyses)
     const rAnalyses = _.filter(({ name }) => _.includes(getExtension(name), tools.RStudio.ext), rawAnalyses)
 
-    //we map the `toolLabel` and `updated` fields to their corresponding header label, which simplifies the table sorting code
-    const enhancedNotebooks = _.map(notebook => _.merge(notebook, { application: tools.Jupyter.label, lastModified: new Date(notebook.updated).getTime() }), notebooks)
-    const enhancedRmd = _.map(rAnalysis => _.merge(rAnalysis, { application: tools.RStudio.label, lastModified: new Date(rAnalysis.updated).getTime() }), rAnalyses)
+    //we map the `toolLabel` corresponding header label, which simplifies the table sorting code
+    const enhancedNotebooks = _.map(notebook => _.merge(notebook, { application: tools.Jupyter.label }), notebooks)
```

Comment:
Now that there's only one item in the merged object, this could possibly be simplified to something like:

```suggestion
    const enhancedNotebooks = _.map(_.set('application', tools.Jupyter.label), notebooks)
```

Same comment for `enhancedRmd` and `enhancedAnalyses`.